docs/html/development/getting-started.rst

```diff
@@ -129,6 +129,11 @@ To build it locally, run:
 
 The built documentation can be found in the ``docs/build`` folder.
 
+.. note::
+   For each Pull Request made the documentation is deployed following this link
+   ``http://pip--<PR-NUMBER>.org.readthedocs.build/en/<PR-NUMBER>``.
```

Comment:
It might help readability to put this in a code block instead of inline. Also I believe HTTPS works for this link.